### ISSUE TITLE
Menu buttons should have a better indicator of interactiveness

### ISSUE THREAD

**Issue by VirginiaBalseiro** [dokieli/dokieli#504]
Color cannot be the only way to indicate that an element is interactive.
This is because users with color blindness or low vision may not be able to perceive the difference.

See: https://www.w3.org/WAI/WCAG22/Understanding/use-of-color
